overlord/hookstate/hooks_test.go

```diff
@@ -67,12 +67,6 @@ func (s *gateAutoRefreshHookSuite) SetUpTest(c *C) {
 	s.state.Lock()
 	defer s.state.Unlock()
 
-	// disable refresh-app-awareness (it's enabled by default);
-	// specific tests below enable it back.
-	tr := config.NewTransaction(s.state)
-	tr.Set("core", "experimental.refresh-app-awareness", false)
-	tr.Commit()
```

Comment:
do we strictly need to remove this?

Reply:
Excellent point, I put it back and just updated the comment